user_docs/en/changes.t2t

```diff
@@ -35,7 +35,7 @@ There are now gestures for showing the braille settings dialog, accessing the st
 - When using a Braille Display via the Standard HID braille driver, the dpad can be used to emulate the arrow keys and enter. Also space+dot1 and space+dot4 now map to up and down arrow respectively. (#14713)
 - Script for reporting the destination of a link now reports from the caret / focus position rather than the navigator object. (#14659)
 - Portable copy creation no longer requires that a drive letter be entered as part of the absolute path. (#14681)
--
+- Gestures for the Eurobraille braille displays are updated. Please refer to the NVDA user guide for an overview. 

```

Comment:
Please remove the changes in this file:

```suggestion
- 
```

First, as written in the [Pull Request Template Explanation And Examples](https://github.com/nvaccess/nvda/blob/master/devDocs/githubPullRequestTemplateExplanationAndExamples.md) in the paragraph describing the "Change log entries" section:
> Because the `changes.t2t` file is prone to conflicts, we ask contributors not to edit the file directly, but instead add the entry to the bottom of the PR description.
> A lead developer will update file when merging the pull request.

Moreover, a second point (not applicable anymore however when you have removed the changes in this file). But keep in mind that in t2t syntax, the dash at the end of a list serve to close it.